roles/memory/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: capture meminfo
+  slurp:
+    src: /proc/meminfo
+  register: meminfo_b64
+
+- name: Set collected meminfo as facts
+  set_fact:
+    stockpile_meminfo:
+      meminfo: "{{ meminfo_b64.content | b64decode }}"
```

Comment:
why are you b64encoding this?

Reply:
The slurp module fetchs the content a base64 encoded file so I decode it before setting the fact

Reply:
I missed the slurp! Thanks!!!